README.md

```diff
@@ -118,6 +118,7 @@ You need the following permissions to run this module.
 
 - [ Basic example](examples/basic)
 - [ Complete example with BYOK encryption and CBR rules](examples/complete)
```

Comment:
Can you update the title in the complete example README.md to "Complete example with topics and schema creation, BYOK encryption and CBR rules"

Reply:
Done.